<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UO1OTNH" failures="1" tests="1" name="Parabank.verifybrokenlinksparabank" time="41.676" errors="0" timestamp="2024-08-08T21:12:18 IST" skipped="0">
  <testcase classname="Parabank.verifybrokenlinksparabank" name="checkBrokenLinks" time="41.676">
    <failure message="Broken link found: https://support.cloudflare.com/hc/en-us/articles/200171906-Error-522 - Response Code: 403" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Broken link found: https://support.cloudflare.com/hc/en-us/articles/200171906-Error-522 - Response Code: 403
at org.testng.Assert.fail(Assert.java:110)
at Parabank.verifybrokenlinksparabank.checkLink(verifybrokenlinksparabank.java:76)
at Parabank.verifybrokenlinksparabank.checkBrokenLinks(verifybrokenlinksparabank.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- checkBrokenLinks -->
  <system-out>
    <![CDATA[Navigating to the Parabank page
Fetching all links on the page
Valid link: https://www.cloudflare.com/5xx-error-landing?utm_source=errorcode_522&utm_campaign=parabank.parasoft.com - Response Code: 200
Valid link: https://www.cloudflare.com/5xx-error-landing?utm_source=errorcode_522&utm_campaign=parabank.parasoft.com - Response Code: 200
Valid link: https://www.cloudflare.com/5xx-error-landing?utm_source=errorcode_522&utm_campaign=parabank.parasoft.com - Response Code: 200
Broken link found: https://support.cloudflare.com/hc/en-us/articles/200171906-Error-522 - Response Code: 403]]>
  </system-out>
</testsuite> <!-- Parabank.verifybrokenlinksparabank -->
